src/components/IconSpinner/index.tsx

```diff
@@ -0,0 +1,27 @@
+import {
+  Flex,
+  Icon,
+  Spinner as ChakraSpinner,
+  useMultiStyleConfig,
+} from "@chakra-ui/react"
+import { FC } from "react"
+import { IconType } from "react-icons"
+
+export interface IconSpinnerProps {
+  icon?: IconType
+}
+
+export const IconSpinner: FC<IconSpinnerProps> = ({ icon }) => {
+  const styles = useMultiStyleConfig("IconSpinner", {})
+  return (
+    <Flex sx={styles.iconSpinnerWrapper}>
+      <ChakraSpinner
+        thickness="8px"
+        speed="1.3s"
+        emptyColor="gray.200"
```

Comment:
If you agree with the comment above:
```
<ChakraSpinner
        thickness="8px"
        speed="1.3s"
        emptyColor="gray.200"
        {...restSpinnerProps}
        sx={styles.chakraSpinner}
```

Reply:
affebc55c25fd7cceeaba54b9b0902bb26334202